axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceLineServiceImpl.java

```diff
@@ -542,6 +542,17 @@ private InvoiceLine computeAnalyticDistributionWithUpdatedQty(InvoiceLine invoic
     return invoiceLine;
   }
 
+  @Override
+  public boolean checkAnalyticDistribution(InvoiceLine invoiceLine) {
+    return invoiceLine == null
+        || CollectionUtils.isNotEmpty(invoiceLine.getAnalyticMoveLineList())
+            && invoiceLineAnalyticService.validateAnalyticMoveLines(
```

Comment:
Add some brackets please, to make it easier to understand and to be sure the condition is well evaluated